code/modules/clothing/shoes/colour.dm

```diff
@@ -109,5 +109,16 @@
 		var/mob/living/carbon/human/C = user
 		if(C.shoes == src && chained == 1)
 			to_chat(user, "<span class='warning'>You need help taking these off!</span>")
+			return FALSE
+	return ..()
+
+/obj/item/clothing/shoes/sneakers/orange/MouseDrop(atom/over)
+	var/mob/m = usr
+	if(ishuman(m))

```

Comment:
You'll only be doing something here if `chained` is true anyway so check it before this type check.
It's only ever 1 or 0 aka TRUE or FALSE so do
`if(chained && ishuman(m))`